packages/block-editor/src/components/block-list/use-block-props/use-zoom-out-mode-exit.js

```diff
@@ -29,6 +33,14 @@ export function useZoomOutModeExit( { editorMode } ) {
 			function onDoubleClick( event ) {
 				if ( ! event.defaultPrevented ) {
 					event.preventDefault();
+
+					const { __experimentalSetIsInserterOpened } = getSettings();
+
+					if (
+						typeof __experimentalSetIsInserterOpened === 'function'
+					) {
+						__experimentalSetIsInserterOpened( false );
+					}
```

Comment:
This pattern follows the advice in https://github.com/WordPress/gutenberg/pull/64573#discussion_r1751810383